spec/fixtures/sshd_config_sles_12_x86_64

```diff
@@ -0,0 +1,139 @@
+# This file is being maintained by Puppet.
```

Comment:
This introduces a change in behavior to suse 12. Not sure why this is part of fixing a bug for Solaris.

``` diff
$ diff -Naur sshd_config_suse_x86_64 sshd_config_sles_12_x86_64
--- sshd_config_suse_x86_64	2016-04-04 15:56:34.000000000 -0400
+++ sshd_config_sles_12_x86_64	2016-05-17 11:57:40.000000000 -0400
@@ -135,5 +135,5 @@
 XAuthLocation /usr/bin/xauth

 # override default of no subsystems
-Subsystem sftp /usr/lib64/ssh/sftp-server
+Subsystem sftp /usr/lib/ssh/sftp-server
```